idl/src/main/proto/todo.proto

```diff
@@ -21,7 +21,15 @@ message TaskResponse {
     string taskId = 1;
 }
 
+message ListPanelsRequest {}
+
+message ListPanelsResponse {
+    string panelId = 1;
+    string title = 2;
+}
+
 service PanelResource {
     rpc AddPanel (PanelRequest) returns (PanelResponse) {}
     rpc AddTask (TaskRequest) returns (TaskResponse) {}
+    rpc ListPanels (ListPanelsRequest) returns (stream ListPanelsResponse) {}
```

Comment:
Without `stream`, generated classes are the same, everything seems fine, but you cannot send more than one response. Instead, you must send a summary (only an `onNext` invocation is allowed).

To solve the problem, just add `stream` and you will be able to send multiple events in the same stream (more than one `onNext`).